helm-chart/todoapp/charts/mysql/templates/statefulSet.yml

```diff
@@ -0,0 +1,89 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Release.Name }}-{{ .Chart.Name }}
+  namespace: {{ .Values.common.namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Chart.Name }}
+  replicas: {{ .Values.mysql.statefulSet.replicas }}
```

Comment:
Verify that `{{ .Values.mysql.statefulSet.replicas }}` is defined in your `values.yaml` file. If not set, this could cause the StatefulSet to not deploy the desired number of replicas.